analysis/import/SCMLogParser/src/main/java/de/maibornwolff/codecharta/importer/scmlogparser/SCMLogParser.java

```diff
@@ -0,0 +1,62 @@
+package de.maibornwolff.codecharta.importer.scmlogparser;
+
+import de.maibornwolff.codecharta.importer.scmlogparser.parser.GitLogParserStrategy;
+import de.maibornwolff.codecharta.importer.scmlogparser.parser.LogParser;
+import de.maibornwolff.codecharta.importer.scmlogparser.parser.LogParserStrategy;
+import de.maibornwolff.codecharta.importer.scmlogparser.parser.SVNLogParserStrategy;
+import de.maibornwolff.codecharta.model.Project;
+import de.maibornwolff.codecharta.serialization.ProjectSerializer;
+
+import java.io.IOException;
+
+public class SCMLogParser {
+
+    public static void main(String[] args) throws IOException {
+        if (args.length >= 1) {
+            if (args[0].equals("-help") || args[0].equals("--help")) {
```

Comment:
it would be cool to be able to directly pipe output from git log to scmlogparser